server/service/src/sync/sync_on_central/mod.rs

```diff
@@ -1,19 +1,29 @@
+use std::{sync::RwLock, vec};
+
 use repository::{ChangelogRepository, SyncBufferRowRepository};
 use util::format_error;
 
 use crate::{
     service_provider::ServiceProvider,
-    sync::{api::SyncApiV5, translations::ToSyncRecordTranslationType, CentralServerConfig},
+    sync::{
+        api::SyncApiV5,
+        api_v6::{SiteStatusCodeV6, SiteStatusV6},
+        synchroniser::integrate_and_translate_sync_buffer,
```

Comment:
It feels like I'm doing something with layers wrong importing the synchroniser method here to do the integration.... should it at least live in a separate file and be used in the two places??